saga/sagalogs/file_saga_log.go

```diff
@@ -0,0 +1,387 @@
+package sagalogs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/scootdev/scoot/saga"
+)
+
+// Writes Saga Log to file system.  Not durable beyond machine failure
```

Comment:
because if the hard drive fails, the file logs aren't replicated to another machine.  So they are lost forever.  

In our final solution using Manhattan, the log is replicated 3 times so it'll survive 2 machine failures